lib/kube/proxy/auth_test.go

```diff
@@ -278,8 +278,10 @@ current-context: foo
 		},
 	}
 	for _, tt := range tests {
+		tt := tt
```

Comment:
Out of curiosity, what's the purpose of this? Is it a workaround for loop memory aliasing or something?

Reply:
The loop variable `tt` is by default captured by reference and all the goroutines spawn by `t.Run` will receive the same reference to `tt`.
Once all tests execute, all of them will referer to the same reference and run the same test.

Using this, we create a new in-loop variable `tt` and the test will capture that variable instead of the old one. With `tt:=tt` we allocate new stack space and when the closure captures the reference, it captures the new allocated space reference, which is different from test to test.